include/deal.II/base/partitioner.templates.h

```diff
@@ -675,17 +687,9 @@ namespace Utilities
             {
               for (auto const &import_indices_plain : import_indices_plain_dev)
                 {
+                  // We can't easily assert here, so we just move the pointer
+                  // matching the host code.
```

Comment:
I don't get the comment. Compared to the CPU code, we removed an assert for the GPU code? You say we can't do the assert but we are not asserting anything right now.

Reply:
In the insert case, nothing should happen with the ghosts as the owner is supposed to set them (and we do not keep track of individual set entries). So I think this is correct.

Reply:
I think what @Rombur was trying to point out is that the "We can't easily assert here" comment seems to refer to some other place in the code base where we *can* easily assert, but that that reference is lost on the casual reader. Maybe saying what one would *want* to assert would have helped, or an "In contrast to the same code in the xxx function". :-)